azureeyemodule/app/model/objectdetector.cpp

```diff
@@ -90,6 +90,58 @@ bool ObjectDetector::pull_data(cv::GStreamingCompiled &pipeline)
     return true;
 }
 
+bool ObjectDetector::pull_data_uvc_video(cv::GStreamingCompiled &pipeline)
```

Comment:
This seems like a lot of code duplication. Are you sure you can't combine this method with the normal pull_data method? Don't force it if you think it will result in uglier/harder-to-maintain code though.

Reply:
![image](https://user-images.githubusercontent.com/13621569/124130207-1b7a5280-dab1-11eb-9544-cda8c2cb3f2e.png)
To give you more context, the flow didn't change a lot from the default one to this uvc_video one but the elements of cv::gout() did. One reason is because for UVC and video, G-API doesn't generate any h264 outputs. So I prefer to keep it as current implementation. What do you think?